recipes/apply_defaults/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "apply_defaults" %}
```

Comment:
The package's name is ``apply-defaults`` on ``pypi``, I think it would be good to follow the same thing